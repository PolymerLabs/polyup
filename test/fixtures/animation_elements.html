<link rel='import'
      href='components/core-animated-pages/core-animated-pages.html'>
<link rel='import'
      href='components/core-animated-pages/transitions/slide-from-right.html'>
