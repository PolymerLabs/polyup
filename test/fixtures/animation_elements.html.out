<link rel="import" href="components/neon-animation/neon-animated-pages.html">
<link rel="import" href="components/neon-animation/animations/slide-from-right-animation.html">
